ansible/library/port_alias.py

```diff
@@ -112,7 +112,7 @@ def get_portmap(self):
                     aliases.append(alias)
                     portmap[name] = alias
                     aliasmap[alias] = name
-                    if speed_index != -1:
+                    if (speed_index != -1) and (len(mapping) > speed_index):
```

Comment:
>(speed_index != -1) and (len(mapping) > speed_index) [](start = 23, length = 52)

speed_index in mapping

Reply:
speed_index is just an index, not a string, so it couldn't be a member of mapping. we can not use membership operator here.